packages/ember-glimmer/tests/integration/application/actions-test.js

```diff
@@ -48,3 +49,25 @@ moduleFor('Application test: actions', class extends ApplicationTest {
       });
   }
 });
+
+moduleFor('Rendering test: non-interactive actions', class extends RenderingTest {
+  getBootOptions() {
+    return { isInteractive: false };
+  }
+
+  [`@test doesn't attatch actions`](assert) {
+    this.registerComponent('foo-bar', {
+      ComponentClass: Component.extend({
+        actions: {
+          fire() {}
```

Comment:
Can you add an assert.ok(false) and try to trigger a click on the button? Just to doubly ensure that nothing happens...